content/reference/sql/version-control/dolt-sql-functions.md

```diff
@@ -359,7 +471,7 @@ They will be removed in a future release.
 
 ## `DOLT_ADD()`
 
-Deprecated. Use the [DOLT\_ADD stored
+Deprecated. Use the [DOLT_ADD stored
```

Comment:
These seem to render and work correctly on prod with the escaped `_`. Looks like we just don't need to escape that at all?

Reply:
We use non-escaped `_` elsewhere and they render correctly. Will double check on dev before I go to prod